File: Python/algorithms/linearSearch.py
def linearSearch(array, value):

    for i in array:
        if i == value:
            return True

    return False

array1 = [1,2,3,5,6,7,8]
array2 = [1,7,8,5,9,3,4]

print("value found") if (linearSearch(array1, 4)) else print("value NOT found")
print("value found") if (linearSearch(array2, 7)) else print("value NOT found")

File: Java/algorithms/linearSearch.java
class linearSearch {
    public static boolean linearS(int[] array, int value) {
        for (int i = 0; i < array.length; i++) {
            if (array[i] == value) {
                return true;
            }
        }
        return false;

    }

    public static void main(String[] args) {
        int[] array1 = { 3, 6, 2, 5, 7, 1, 9 };
        int[] array2 = { 4, 9, 1, 7, 6, 3, 8 };

        if (linearS(array1, 7)) {
            System.out.println("Found!");
        } else {
            System.out.println("Not Found");
        }

        if (linearS(array2, 5)) {
            System.out.println("Found!");
        } else {
            System.out.println("Not Found");
        }
    }
}
